src/compass/s1_geocode_slc.py

```diff
@@ -195,6 +196,95 @@ def run(cfg: GeoRunConfig):
                                    weather_model_path=cfg.weather_model_file,
                                    delay_type=cfg.tropo_params.delay_type)
 
+            # Designate radiometric calibration parameter to geocode
+            calibration_dict = {
+                'gamma':burst.burst_calibration.gamma,
```

Comment:
Please note that `beta_naught` is not provided here because it is constant, and was already provided in `/science/SENTINEL1/CSLC/corrections/calibration`